application/src/test/java/org/togetherjava/tjbot/features/basic/SlashCommandEducatorTest.java

```diff
@@ -65,6 +65,6 @@ private static Stream<String> provideMessageCommands() {
     }
 
     private static Stream<String> provideOtherMessages() {
-        return Stream.of("  a  ", "foo", "#foo", "/foo", "!!!", "?!?!?", "?", ".,-", "!f", "! foo");
+        return Stream.of("  a  ", "foo", "#foo", "/foo", "!!!", "?!?!?", "?", ".,-", "!f", "! foo","thisIsAWordWhichLengthIsMoreThanThrityLetterSoItShouldNotReply");
```

Comment:
it doesn't have to be in a single work tho, also make another one which starts with a command prefix